allensdk/brain_observatory/behavior/write_behavior_nwb/__main__.py

```diff
@@ -0,0 +1,73 @@
+import os
+import logging
+import sys
+import argschema
+import marshmallow
+
+from allensdk.brain_observatory.behavior.behavior_session import (
+    BehaviorSession)
+from allensdk.brain_observatory.behavior.session_apis.data_io import (
+    BehaviorNwbApi, BehaviorJsonApi)
+from allensdk.brain_observatory.behavior.write_behavior_nwb._schemas import (
+    InputSchema, OutputSchema)
+from allensdk.brain_observatory.argschema_utilities import (
+    write_or_print_outputs)
+from allensdk.brain_observatory.session_api_utils import sessions_are_equal
+
+
+def write_behavior_nwb(session_data, nwb_filepath):
```

Comment:
This module is identical to `write_nwb.__main__`

Ideally, they would both inherit from the same base class and share logic.